### ISSUE TITLE
Not compiling on iOS

### ISSUE THREAD

**Issue by alexp25** [elevationsoftware/cordova-plugin-camera-stream#4]
I am trying to use this plugin but it has deprecated references, ```cordova-plugin-add-swift-support``` should be ```^2.0.2``` and UIWebView is deprecated, should switch to WKWebView, some functions are deprecated as well, etc. It seems that it's the only plugin that works to bring camera stream to canvas on iOS, so please fix it, as I think there are just minor compatibility issues.

**Comment by pirumpi** [comment]
What version of iOS are you using?

**Comment by alexp25** [comment]
I am running iOS 13.6.1. Here is the output of ```ionic info```:
```
Ionic:

   Ionic CLI                     : 5.4.15 (/usr/local/lib/node_modules/ionic)
   Ionic Framework               : @ionic/angular 4.9.0
   @angular-devkit/build-angular : 0.801.3
   @angular-devkit/schematics    : 8.1.3
   @angular/cli                  : 8.3.5
   @ionic/angular-toolkit        : 2.0.0

Cordova:

   Cordova CLI       : 9.0.0 (cordova-lib@9.0.1)
   Cordova Platforms : ios 5.1.1
   Cordova Plugins   : cordova-plugin-ionic-keyboard 2.2.0, cordova-plugin-ionic-webview 4.2.1, (and 48 other plugins)

Utility:

   cordova-res (update available: 0.15.1) : 0.6.0
   native-run (update available: 1.0.0)   : 0.2.8

System:

   ios-deploy : 1.9.4
   ios-sim    : 8.0.2
   NodeJS     : v10.16.0 (/usr/local/bin/node)
   npm        : 6.9.0
   OS         : macOS Catalina
   Xcode      : Xcode 11.6 Build version 11E708
```
